cli/cmd/config.go

```diff
@@ -60,10 +66,144 @@ func setupCommand(options ...setupOption) func(cmd *cobra.Command, args []string
+						{Header: "ENABLED", Path: "spec.enabled"},
+						{Header: "MINIMUM SCORE", Path: "spec.minimumScore"},
+					},
+				},
+			),
+		)
+
+		resources.Register(
+			resourcemanager.NewClient(
+				httpClient,
+				"pollingprofile", "pollingprofiles",
+				resourcemanager.TableConfig{
+					Cells: []resourcemanager.TableCellConfig{
+						{Header: "ID", Path: "spec.id"},
+						{Header: "NAME", Path: "spec.name"},
+						{Header: "STRATEGY", Path: "spec.strategy"},
+					},
+				},
+			),
+		)
+
+		resources.Register(
+			resourcemanager.NewClient(
+				httpClient,
+				"demo", "demos",
+				resourcemanager.TableConfig{
+					Cells: []resourcemanager.TableCellConfig{
+						{Header: "ID", Path: "spec.id"},
+						{Header: "NAME", Path: "spec.name"},
+						{Header: "TYPE", Path: "spec.type"},
+						{Header: "ENABLED", Path: "spec.enabled"},
+					},
+				},
+			),
+		)
+
+		resources.Register(
+			resourcemanager.NewClient(
+				httpClient,
+				"datastore", "datastores",
+				resourcemanager.TableConfig{
+					Cells: []resourcemanager.TableCellConfig{
+						{Header: "ID", Path: "spec.id"},
+						{Header: "NAME", Path: "spec.name"},
+						{Header: "DEFAULT", Path: "spec.default"},
+					},
+					ItemModifier: func(item *gabs.Container) {
+						isDefault := item.Path("spec.default").Data().(bool)
+						if !isDefault {
+							item.SetP("", "spec.default")
+						} else {
+							item.SetP("*", "spec.default")
+						}
+					},
+				},
+			),
+		)
+
+		resources.Register(
+			resourcemanager.NewClient(
+				httpClient,
+				"environment", "environments",
+				resourcemanager.TableConfig{
+					Cells: []resourcemanager.TableCellConfig{
+						{Header: "ID", Path: "spec.id"},
+						{Header: "NAME", Path: "spec.name"},
+						{Header: "DESCRIPTION", Path: "spec.description"},
+					},
+				},
+			),
+		)
+
+		resources.Register(
+			resourcemanager.NewClient(
+				httpClient,
+				"transaction", "transactions",
+				resourcemanager.TableConfig{
+					Cells: []resourcemanager.TableCellConfig{
+						{Header: "ID", Path: "spec.id"},
+						{Header: "NAME", Path: "spec.name"},
+						{Header: "VERSION", Path: "spec.version"},
+						{Header: "STEPS", Path: "spec.summary.steps"},
+						{Header: "RUNS", Path: "spec.summary.runs"},
+						{Header: "LAST RUN TIME", Path: "spec.summary.lastRun.time"},
+						{Header: "LAST RUN SUCCESSES", Path: "spec.summary.lastRun.passes"},
+						{Header: "LAST RUN FAILURES", Path: "spec.summary.lastRun.fails"},
+					},
+					ItemModifier: func(item *gabs.Container) {
+						// set spec.summary.steps to the number of steps in the transaction
+						item.SetP(len(item.Path("spec.steps").Children()), "spec.summary.steps")
+
+						// if lastRun.time is not empty, show it in a nicer format
+						lastRunTime := item.Path("spec.summary.lastRun.time").Data().(string)
+						if lastRunTime != "" {
+							date, err := time.Parse(time.RFC3339, lastRunTime)
+							if err != nil {
+								panic(err)
```

Comment:
Should we panic here or only emit a warning log and set the time as empty?

Reply:
good catch! I changed it to return an error!